test/validation.test.ts

```diff
@@ -54,4 +55,96 @@ describe('Unit tests on field validation', () => {
       expect(value).toEqual('+33140506070');
     });
   });
+
+  describe('redirections', () => {
```

Comment:
It seems that we're only checking valid URLs? If that's correct, I think we should also have tests for invalid URLs.

Reply:
yes indeed